devito/symbolics/manipulation.py

```diff
@@ -17,7 +18,7 @@
 
 __all__ = ['xreplace_indices', 'pow_to_mul', 'indexify', 'subs_op_args',
            'normalize_args', 'uxreplace', 'Uxmapper', 'reuse_if_untouched',
-           'evalrel']
+           'evalrel', 'flatten_args']
```

Comment:
Why not move this if it causes a circular import?

Reply:
1) this is where it belongs
2) I would need a local important in this file otherwise